extensions/dicom-pdf/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@ohif/extension-dicom-pdf",
```

Comment:
There is already a published package for `@ohif/extension-dicom-pdf` for v2, this is actually true for other extensions as well. We are not publishing to npm until v3 is on master? what happens to the old v2 extensions then? @swederik 
It think we should do better versioning of these extensions as well. A major version release of OHIF should reset everyone to the version base number ...

Reply:
If we need to have different versioning, then probably using @ohif3/... would be better, as then you can still release v2 ones, or else the major version of the release could be 3.x, and the rest could be the individual version stuff.